lib/datadog/tracing/contrib/sinatra/patcher.rb

```diff
@@ -53,7 +40,6 @@ def patch
             register_tracer
 
             patch_default_middlewares
-            setup_tracer
           end
 
           def register_tracer
```

Comment:
:arrow_down: Don't forget to remove `setup_tracer`, below

Reply:
Good catch, removed.